src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_rsa.cpp

```diff
@@ -7,8 +7,8 @@
 static int32_t ExecuteCFDataTransform(
     SecTransformRef xform, uint8_t* pbData, int32_t cbData, CFDataRef* pDataOut, CFErrorRef* pErrorOut);
 
-extern "C" int
-AppleCryptoNative_RsaGenerateKey(int32_t keySizeBits, SecKeyRef* pPublicKey, SecKeyRef* pPrivateKey, int32_t* pOSStatus)
+extern "C" int AppleCryptoNative_RsaGenerateKey(
```

Comment:
Fixed these two `int` => `int32_t`s.  There are others, which is causing clang-format to have a conniption; I'll do that as a style only change after this one goes in.